modules/govuk_cdnlogs/templates/etc/logrotate.cdn_logs_hourly.conf.erb

```diff
@@ -4,7 +4,7 @@
 -%>
 
 {
-  rotate 720
+  hourly
```

Comment:
I think this should be added in addition to `rotate 720`. I think we still require the number to rotate on, but also need to specify the `hourly` parameter.